app/views/grammars/_form.html.erb

```diff
@@ -13,7 +13,7 @@
 
   <div class="field">
     <%= form.label :dialect_id %>
-    <%= form.text_field :dialect_id %>
+    <%= form.select :dialect_id, Dialect.all.pluck(:name_en, :id) %>
```

Comment:
Maybe we should sort this alphabetically? What do you think?

Reply:
Checkout the rails `order` method. https://apidock.com/rails/ActiveRecord/QueryMethods/order

Reply:
Done.